src/utils/context.rs

```diff
@@ -114,6 +114,8 @@ pub struct Context {
     pub is_main_ctx: bool,
     /// Determines if the context is in an unsafe block
     pub is_unsafe_ctx: bool,
+    /// Determines if text should be escaped e.g. for file globs
+    pub is_escaped_ctx: bool,
```

Comment:
this is interesting, I think that can be used for:

* https://github.com/amber-lang/amber/pull/376
* https://github.com/amber-lang/amber/pull/381
* https://github.com/amber-lang/amber/issues/270